features/steps/ios_steps.rb

```diff
@@ -1,3 +1,13 @@
+When("I run {string} with the defaults") do |eventType|
```

Comment:
I'd change this to pass through the simulator value as well, as that is how the tests will likely be split in the near future. One option:

```ruby
When("I run {string} on the {string}") do |event_type, simulator|
```